xblockgrademe/xblockgrademe.py

```diff
@@ -25,15 +25,29 @@ def workbench_scenarios():
             ('Xblock GradeMe',
              """<sequence_demo>
                     <xblockgrademe />
-                    <xblockgrademe name="My First XBlock" />
                 </sequence_demo>
              """),
         ]
 
-    name = String(
-        default='Xblock GradeMe',
+    display_name = String(
+        default="'Grade Me' Button",
```

Comment:
This is actually grammatically incorrect; quoted strings of English text
must use double quotes in presentation.

```python
default='"Grade Me" Button'
```

so that it's displayed as `"Grade Me" Button`.